src/main/java/appeng/client/gui/implementations/GuiOreFilter.java

```diff
@@ -164,9 +165,19 @@ public void setTextFieldValue(final String displayName, final int mousex, final
         final int[] ores = OreDictionary.getOreIDs(stack);
 
         if (ores.length > 0) {
-            textField.setText(OreDictionary.getOreName(ores[0]));
+
+            if (Mouse.isButtonDown(0)) {
+                // left click
+                textField.setText(textField.getText() + OreDictionary.getOreName(ores[0]));
```

Comment:
maybe it's worth adding a separator ` | ` if `textField.getText()` is not empty?